packages/sdks/python/tester-local/pyproject.toml

```diff
@@ -6,9 +6,9 @@ authors = ["Your Name <you@example.com>"]
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = "^3.11"
+python = "^3.13"
```

Comment:
This python lib should still work for 3.11 and 3.12 right? ^3.11 means 3.11 and all newer minor versions (i.e. also 3.12, 3.13, etc)

Reply:
I believe it should yea

Reply:
In that case keep the ^3.11, otherwise people with older versions of python will be told by installers like poetry that its not compatible with their version